scripts/ci/version.sh

```diff
@@ -22,17 +22,17 @@ platform=`uname`
 
 for each in $(find src -name __init__.py); do
     if [ "$platform" == "Darwin" ]; then
-        sed -i "" "s/^__version__ = [\"']\(.*\)+dev[\"']/__version__ = \"\1+dev.$version\"/" $each
+        sed -i "" "s/^__version__ = [\"']\(.*\)[\"']/__version__ = \"\1.$version\"/" $each
     else
-        sed -i "s/^__version__ = [\"']\(.*\)+dev[\"']/__version__ = \"\1+dev.$version\"/" $each
+        sed -i "s/^__version__ = [\"']\(.*\)[\"']/__version__ = \"\1.$version\"/" $each
     fi
 done
 
 for each in $(find src -name setup.py); do
     if [ "$platform" == "Darwin" ]; then
-        sed -i "" "s/^VERSION = [\"']\(.*\)+dev[\"']/VERSION = \"\1+dev.$version\"/" $each
+        sed -i "" "s/^VERSION = [\"']\(.*\)[\"']/VERSION = \"\1.$version\"/" $each
     else
-        sed -i "s/^VERSION = [\"']\(.*\)+dev[\"']/VERSION = \"\1+dev.$version\"/" $each
+        sed -i "s/^VERSION = [\"']\(.*\)[\"']/VERSION = \"\1.$version\"/" $each
```

Comment:
This is used to generate edge build. With the `+dev` gone, we need to make sure the version scheme still works for edge build.

Reply:
Yes. Can you take a look at this since you know about this? Is there something I can do to verify?

Reply:
use `.devN`

For example.

The version in the source code is `2.0.19`.
The nightly version become `2.0.19.dev4345`. The digits follows `dev` is Travis build number.
After release, the latest package in PyPI is `2.0.19`. In the meantime, source code version is upgraded to `2.0.20`.

Reply:
Reference: https://www.python.org/dev/peps/pep-0440/#public-version-identifiers

Reply:
okay done.